sdk/component/store/remote/postgresql/sqlc/migrations/20241209103854_findings_table.sql

```diff
@@ -0,0 +1,8 @@
+-- create "finding" table
+CREATE TABLE IF NOT EXISTS finding (
+   id SERIAL PRIMARY KEY,
+   instance_id UUID NOT NULL,
+   details JSONB NOT NULL,
+   created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+   updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
```

Comment:
You might already have this somewhere, but can we have an index on instance_id?

Reply:
Tracked here https://linear.app/smithy/issue/OCU-279/add-database-indexes-for-findings-table